lib/handlers_v5.js

```diff
@@ -341,3 +344,21 @@ function shutdown_request(request) {
 
     status_idle.call(this, request);
 }
+
+function is_complete_request(request) {
```

Comment:
Then again (sorry, thinking out loud) I don't develop in coffeescript, but it seems to me that the need for good multiline editing support is even stronger there because of the significant whitespace. So assuming that there is a coffeescript parser that is up to the task (perhaps the jison parser that coffeescript itself uses), my suggestion of putting the parse function separately in jp-babel, jp-javascript, and jp-coffeescript might be the most robust option.